cmdc/client.py

```diff
@@ -139,7 +177,28 @@ def __init__(self, apikey=None):
             )
             print(msg)
 
-        pass
+    ## county/state map
+    @property
+    def counties(self):
+        if self._counties is None:
+            # fetch
+            res = self.sess.get(BASE_URL + "/counties")
+            if not res.ok:
+                raise ValueError("Failed to get list of counties")
+
+            self._counties = pd.DataFrame(res.json())
+            self._counties["state"] = (
+                self._counties["fips"].astype(str).str.zfill(5).str[:2].astype(int)
+            )
+
+        return self._counties
+
+    def fips_in_states(self, states: Union[List[int], int]):
+        if isinstance(states, (int, str)):
+            states = [states]
+        states = [int(us.states.lookup(str(x)).fips) for x in states]
```

Comment:
This fails for single digit numbers (CA is 6 for instance and the lookup fails)

```suggestion
        states = [int(us.states.lookup(str(x).zfill(2)).fips) for x in states]
```